src/test/java/co/com/api/test/dto/GithubRepoListDto.java

```diff
@@ -0,0 +1,25 @@
+package co.com.api.test.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Builder
+@EqualsAndHashCode
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
why do you need to create a private constructor?

Reply:
I add these lines to avoid errors to deserialize from, reading about in a forum, and the recommendation is add this lines, and work correctly. without these lines an error occurs when deserializing

Reply:
but why private?

Reply:
change acceslevel to Public